nautobot/cloud/navigation.py

```diff
@@ -46,6 +46,22 @@
                             ),
                         ),
                     ),
+                    NavMenuItem(
+                        link="cloud:cloudnetwork_list",
+                        name="Cloud Networks",
+                        weight=100,
+                        permissions=[
+                            "cloud.view_cloudnetwork",
+                        ],
+                        buttons=(
+                            NavMenuAddButton(
+                                link="cloud:cloudnetwork_add",
+                                permissions=[
+                                    "cloud.add_cloudnetwork",
+                                ],
+                            ),
+                        ),
+                    ),
```

Comment:
We should think about the ordering of these cloud models on the navigation menu instead of assigning arbitrary weights.
IMO we should order this as the following, but what do you guys think? @nautobot/core 

1. CloudAccount
2. CloudType
3. CloudNetwork
4. CloudService

Reply:
IIRC the general rule we go with is "more specific, more commonly accessed things first" rather than "order of creation/database population". For example:

![image](https://github.com/nautobot/nautobot/assets/5603551/45806716-2dc7-4955-ac85-4103ecdf971f)

So from that standpoint I'd propose:

- CloudNetwork
- CloudService
- CloudType
- CloudAccount